docs/install/any-kubernetes-cluster.md

```diff
@@ -0,0 +1,836 @@
+  - /docs/install/knative-with-aks
+  - /docs/install/knative-with-ambassador
+  - /docs/install/knative-with-contour
+  - /docs/install/knative-with-docker-for-mac
+  - /docs/install/knative-with-gke
+  - /docs/install/knative-with-gardener
+  - /docs/install/knative-with-gloo
+  - /docs/install/knative-with-icp
+  - /docs/install/knative-with-iks
+  - /docs/install/knative-with-microk8s
+  - /docs/install/knative-with-minikube
+  - /docs/install/knative-with-minishift
+  - /docs/install/knative-with-pks
+showlandingtoc: "false"
+---
+
+You can install Knative by applying YAML files using the `kubectl` CLI.
+
+## Prerequisites
+
+Before installation, you must meet the following prerequisites:
+
+- You have a cluster that uses Kubernetes v1.18 or newer.
+- You have installed the [`kubectl` CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+- If you have only one node in your cluster, you will need at least 6 CPUs, 6 GB of memory, and 30 GB of disk storage.
+- If you have multiple nodes in your cluster, for each node you will need at least 2 CPUs, 4 GB of memory, and 20 GB of disk storage.
+<!--TODO: Verify these requirements-->
+- Your Kubernetes cluster must have access to the internet, since Kubernetes needs to be able to fetch images.
+
+## Installing the Serving component
+
+To install the serving component:
+
+1. Install the required custom resources:
+
+   ```bash
+   kubectl apply -f {{< artifact repo="serving" file="serving-crds.yaml" >}}
+   ```
+
+1. Install the core components of Serving (see below for optional extensions):
+
+   ```bash
+   kubectl apply -f {{< artifact repo="serving" file="serving-core.yaml" >}}
+   ```
+
+### Installing a networking layer
+
+Follow the procedure for the networking layer of your choice:
+
+<!-- TODO: Link to document/diagram describing what is a networking layer.  -->
+
+<!-- This indentation is important for things to render properly. -->
+
+   {{< tabs name="serving_networking" default="Istio" >}}
+   {{% tab name="Ambassador" %}}
+
+The following commands install Ambassador and enable its Knative integration.
+
+1. Create a namespace to install Ambassador in:
+
+   ```bash
+   kubectl create namespace ambassador
+   ```
+
+1. Install Ambassador:
+
+   ```bash
+   kubectl apply --namespace ambassador \
+     -f https://getambassador.io/yaml/ambassador/ambassador-crds.yaml \
+     -f https://getambassador.io/yaml/ambassador/ambassador-rbac.yaml \
+     -f https://getambassador.io/yaml/ambassador/ambassador-service.yaml
+   ```
+
+1. Give Ambassador the required permissions:
+
+   ```bash
+   kubectl patch clusterrolebinding ambassador -p '{"subjects":[{"kind": "ServiceAccount", "name": "ambassador", "namespace": "ambassador"}]}'
+   ```
+
+1. Enable Knative support in Ambassador:
+
+   ```bash
+   kubectl set env --namespace ambassador  deployments/ambassador AMBASSADOR_KNATIVE_SUPPORT=true
+   ```
+
+1. To configure Knative Serving to use Ambassador by default:
```

Comment:
```suggestion
1. To configure Knative Serving to use Ambassador by default, enter the command:
```

Reply:
All the procedure steps in this part don't seem to have "run" or "enter" the command. In future I'd maybe keep this as a separate PR to the pingsource stuff, i.e. keep topics separated so it doesn't blur the PR scope